src/ejabberd_c2s_info_handler.erl

```diff
@@ -39,6 +41,17 @@ call(FsmRef, Tag, Data) ->
     FsmRef ! {call_info_handler, Tag, Data},
     ok.
 
+-spec call_after(integer(), jid:jid() | pid(), atom(), term()) -> reference().
```

Comment:
`session_not_found` not found... in the spec

Reply:
Right. @BG, looks like the following would work:
```suggestion
-spec call_after(integer(), jid:jid() | pid(), atom(), term()) -> session_not_found | reference().
```